students/slupek_bartosz/lesson_03_functions/Calculator.py

```diff
@@ -0,0 +1,93 @@
+def multiply():
+    print("MULTIPLY")
+    print("Input 1st operand:")
+    add_var_1 = int(input())
+    print("Input 2nd operand:")
+    add_var_2 = int(input())
+    print("Result:")
+    print(add_var_1 * add_var_2)
+
+
+def add():
+    print("ADDING")
+    print("Input 1st operand:")
+    add_var_1 = int(input())
+    print("Input 2nd operand:")
+    add_var_2 = int(input())
+    print("Result:")
+    print(add_var_1 + add_var_2)
+
+
+def subtract():
+    print("SUBTRACT")
+    print("Input 1st operand:")
+    add_var_1 = int(input())
+    print("Input 2nd operand:")
+    add_var_2 = int(input())
+    print("Result:")
+    print(add_var_1 - add_var_2)
+
+
+def divide():
+    print("DIVIDE")
+    print("Input 1st operand:")
+    add_var_1 = int(input())
+    print("Input 2nd operand:")
+    add_var_2 = int(input())
+    print("Result:")
+    print(add_var_1 / add_var_2)
+
+
+def power():
+    print("POWER")
+    print("Input 1st operand:")
+    add_var_1 = int(input())
+    print("Input 2nd operand:")
+    add_var_2 = int(input())
+    print("Result:")
+    print(add_var_1 ** add_var_2)
+
+
+def help():
+    print("HELP")
+    print("a - add")
+    print("s - subtract")
+    print("m - multiply")
+    print("d - divide")
+    print("p - power")
+    print("h,? - help")
+    print("q - QUIT")
+
+
+def quit():
+    print("GOOD BYE")
+
+
+print("Welcome to badly organized calculator:")
```

Comment:
całą tą część kodu, która już wywołuje kalkulator można by zamknąć w warunku:
`if name == 'main':`
Dzięki czemu podczas importu funkcji z tego modułu do innego modułu/skryptu ta część modułu nie zostanie wykonana.
On się wykona tylko wtedy, gdy ten moduł/skrypt zostanie uruchomiony jak skrypt (na najwyższym poziomie).
Jeżeli uruchomisz ten skrypt zmienna name przyjmie wartość 'main', jeżeli zaimportujesz jako moduł to name ma wartość: 'nazwa_pliku'.
Pamiętaj o tym tworząc nowy moduł i możesz to tez dodać do pozostałych zadań, dzięki czemu robiąc zadania z listy 4 nie będziesz mieć problemów podczas importowaniu funkcji do testowania.
W skryptach  z lekcji 0 użyty był ten mechanizm, możesz zobaczyć jak go użyto.